tests/test_load_quotes.py

```diff
@@ -0,0 +1,25 @@
+from app.database.models import Quote
+from app.internal.quotes import load_quotes
+
+
+def get_quotes_amount(session):
```

Comment:
Maybe add that as part of the table class, and then you can use it in `is_quotes_table_empty` too.
In the tests you can do something like:
```
assert is_quotes_table_empty  # count == 0
assert not is_quotes_table_empty # count > 0
```